tests/helpers/Participant.ts

```diff
@@ -291,7 +309,7 @@ export class Participant {
      */
     async waitToJoinMUC(): Promise<void> {
         return this.driver.waitUntil(
-            () => this.isInMuc(),
+            async () => await this.isInMuc(),
```

Comment:
Is this necessary? It will make the function return a promise, and if `this.isInMuc()` already does that, this change shouldn't be necessary.